src/Pages/Store/Home/Component/Trend/Trend.js

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import "./Trend.scss";
+
+class Trend extends Component {
+  constructor() {
+    super();
+    this.state = {
+      trendList: [],
+      bulletList: [],
+    };
+  }
+
+  getTrendList = () => {
+    fetch("http://localhost:3000/data/trendItem.json", {
+      method: "GET",
+    })
+      .then((res) => res.json())
+      .then((res) => {
+        this.setState({
+          trendList: res.item,
+          bulletList: res.event,
+        });
+      });
+  };
+  componentDidMount() {
+    this.getTrendList();
+  }
+
+  render() {
+    const { trendList, bulletList } = this.state;
+    // console.log(bulletList);
+
+    return (
+      <div className="trendAllContainer">
+        <div className="title">
+          <h2>트렌드 기획전</h2>
+        </div>
+        <div className="trendContainer">
+          <div className="leftContainer">
+            {trendList.map((trend, idx) => {
+              // console.log(trend.bullet);
+              if (idx % 2 === 0) {
```

Comment:
boolean 값을 이용해 아래와 같이 작성할 수 있습니다. 
```suggestion
              if (!idx % 2) {
```